h2/events.py

```diff
@@ -212,6 +267,11 @@ def __init__(self):
         #: the changed settings.
         self.changed_settings = {}
 
+    def __repr__(self):
+        return "<SettingsAcknowledged changed_settings:%s>" % (
+            self.changed_settings,
```

Comment:
That's true, but that's not the settings that are carried on this event: this event carries the settings that were acknowledged, not what was in the SETTINGS frame. =)